app/serializers/npq_application_csv_serializer.rb

```diff
@@ -31,6 +31,7 @@ def csv_headers
       teacher_reference_number
       teacher_reference_number_validated
       school_urn
+      school_ukprn
```

Comment:
Does the order matter here?

If so, do we want to add the new fields to the end of the list, rather than in the middle? I think I heard of some providers relying on indeces of rows rather than names in the header to figure out what's what 😭

Reply:
My understanding is that version 1 is not finalised and subject to changes on our end. I think we should to work to design at least a half decent api. I would deem grouping CSV columns logically to be sensible. If a provider chose to integrate with CSVs programatically and chose to work off indices and not field names then they should understand that implementing in such a fashion would be brittle therefore understood the risks of doing so.

I don't think we should work to design an API due to naive client implementations whilst it is still a work in progress.

Reply:
Fair enough.